### ISSUE TITLE
Cannot build from source as a docker image

### ISSUE THREAD

**Issue by eldos-dl** [performancecopilot/pcp#23]
`./configure` is failing with error while building docker image from Dockerfile
```
checking for ps style... unknown
FATAL ERROR: could not determine how to get the "all processes with arguments"
```
Here is snippet from my Dockerfile

```
FROM debian:wheezy
MAINTAINER <eldos-dl>

# Update and install essentials
RUN apt-get update && apt-get install -y git build-essential wget openssl libssl-dev curl vim pkg-config
# Node installation
RUN wget ftp://ftp.pcp.io/projects/pcp/download/pcp-3.10.4.src.tar.gz
RUN tar xvzf pcp-3.10.4.src.tar.gz && mv pcp-3.10.4 /opt/pcp
# PCP Installation
WORKDIR /opt/pcp/
RUN apt-get install -y flex bison autoconf python2.7-dev python qt4-qmake libmicrohttpd-dev procps
RUN ps -ef
RUN ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var
RUN make && make install
```

**Comment by kmcdonell** [comment]
Could you please post the output from the command below run in the context of the container?
$ ps -ef | awk '$2==1{print}'

**Comment by natoscott** [comment]
Possibly related, in master branch (not yet released)...?

commit cb5f2c1cc9bbf9e65cd0589dcd0f81f79bbac1b9
Author: David Disseldorp <ddiss@suse.de>
Date:   Tue May 5 19:53:18 2015 -0400

    Add configure psproc check for /bin/bash for open build systems

**Comment by eldos-dl** [comment]
Not released in 3.10.4 ftp://ftp.pcp.io/projects/pcp/download/pcp-3.10.4.src.tar.gz

**Comment by kmcdonell** [comment]
Can I please see the ps output I requested earlier?

**Comment by eldos-dl** [comment]
```
root         1     0  0 05:02 ?        00:00:00 /bin/sh -c ps -ef | awk '$2==1{print}'
```
This was the output. Sorry for the delay.

**Comment by natoscott** [comment]
This issue has been resolved in pcp-3.10.6 via completely reworked configure.ac logic in this area.
